style.css

```diff
@@ -365,6 +373,10 @@ footer {
         left: 3.7rem;
     }
 
+    #pageButtons {
+        display: none;
+    }
```

Comment:
Currently the numbered page buttons do not show up on the first page of desktop version, the problem is most likely  here or somewhere else in the css as i can see no major changes to the script.js file

![image](https://github.com/adamkumlin/frontend-image-search/assets/142667693/4820c629-21c6-4370-b250-7707bcc2266c)

Reply:
same, see above